packages/remediations/src/steps/reviewSystems.js

```diff
@@ -0,0 +1,166 @@
+import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { EXISTING_PLAYBOOK, inventoryEntitiesReducer as entitiesReducer, TOGGLE_BULK_SELECT } from '../utils';
+import './reviewSystems.scss';
+
+const ReviewSystems = (props) => {
+    const { issues, systems, registry } = props;
+
+    let dispatch = useDispatch();
+    const inventory = useRef(null);
+    const { input } = useFieldApi(props);
+    const formOptions = useFormApi();
+
+    const [ inventoryApi, setInventoryApi ] = useState();
+
+    const { selected, loaded, rows } = useSelector(({ entities }) => ({
+        selected: entities?.selected || new Map(),
+        loaded: entities?.loaded,
+        rows: entities?.rows || []
+    }));
+
+    const formValues = formOptions.getState().values;
+    const error = formOptions.getState().errors?.systems;
+    const playbook = formValues && formValues[EXISTING_PLAYBOOK];
+
+    const allNewSystems = [ ...new Set(issues.reduce((acc, curr) => [
+        ...acc,
+        ...(curr.systems || [])
+    ], [ ...systems ])) ];
+
+    const allSystems = playbook ? [ ...new Set(playbook.issues?.reduce((acc, curr) => [
+        ...acc,
+        ...(curr.systems?.map(system => system.id) || [])
+    ], [ ...allNewSystems ]))
+    ] : allNewSystems;
+
+    const onRefresh = (options, callback) => {
+        if (!callback && inventory && inventory.current) {
+            inventory.current.onRefreshData(options);
+        } else if (callback) {
+            callback(options);
+        }
+    };
+
+    const onSelect = (selected) => {
+        input.onChange(Array.from(selected.keys()));
+    };
+
+    const onSelectRows = (value) => {
+        dispatch({
+            type: TOGGLE_BULK_SELECT,
+            payload: value
+        });
+    };
+
+    return (
+        <Stack hasGutter>
+            <StackItem>
+                <TextContent>
+                    <Text>
+                        Review and optionally exclude systems from your selection.
+                    </Text>
+                </TextContent>
+            </StackItem>
+            <StackItem>
+                {
+                    <Router>
+                        <InventoryTable
+                            hideFilters={{ all: true }}
+                            noDetail
+                            variant="compact"
+                            showTags
+                            onRefresh={onRefresh}
+                            ref={inventory}
+                            getEntities={async (_i, config) => {
+                                const invData = await inventoryApi?.getEntities(allSystems, {
+                                    ...config,
+                                    hasItems: true
+                                });
+                                const data = await inventoryApi?.mapTags(invData);
```

Comment:
I've checked the inv api and looks like you don't have to fetch the tags if you include 3rd argument `true` to `getEntities`

Reply:
oh, good point, thank you :-)